### PR TITLE
feat(DNA-1937,1938): Alert & Reports Template Changes

### PR DESCRIPTION
**Pull request by SamraHanifCareem**
- [x] Adding type of notification in email and slack
- [x] Removing indentation for slack template
- [x] Adding route to a specific Alert/Reports in email and slack
- [x] Refactoring the template for both the notification methods
- [x] Removing 'Explore in Careem Insights' from msg_content type
- [x] Adding Dynamic base url for Alert/Reports route

### CODE REVIEW

File: tests/integration_tests/celery_tests.py [hunk 1]
```diff
@@ -237,7 +237,9 @@ def test_run_sync_query_cta_config(test_client, ctas_method):
 
     query = get_query_by_id(result["query"]["serverId"])
     assert (
-        sqlparse.format(f"CREATE {ctas_method} {CTAS_SCHEMA_NAME}.{tmp_table_name} AS \n{QUERY}").strip()
+        sqlparse.format(
```
Comment (j4m4l) [inline tests/integration_tests/celery_tests.py:240]: any reason t make these indentation changes ?

File: tests/integration_tests/celery_tests.py [hunk 2]
```diff
@@ -279,7 +281,9 @@ def test_run_async_query_cta_config(test_client, ctas_method):
     )
 
     assert (
-        sqlparse.format(f"CREATE {ctas_method} {CTAS_SCHEMA_NAME}.{tmp_table_name} AS \n{QUERY}").strip()
+        sqlparse.format(
```
Comment (j4m4l) [inline tests/integration_tests/celery_tests.py:284]: any reason to make these indentation changes ?

File: tests/unit_tests/jinja_context_test.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
 from superset.datasets.commands.exceptions import DatasetNotFoundError
 from superset.jinja_context import dataset_macro, where_in
 
+
```
Comment (j4m4l) [inline tests/unit_tests/jinja_context_test.py:28]: any reason to make these indentation changes ?

File: tests/integration_tests/celery_tests.py (hunk 1 above)
Comment (SamraHanifCareem) [reply tests/integration_tests/celery_tests.py:240]: Actually we have used a python formatter 'black' that made these changes. Suggested by 'Superset Community'

**Review by j4m4l — APPROVED** [review]

**Pull request merged**